NotificationService/NotificationService.swift

```diff
@@ -75,10 +70,9 @@ class NotificationService: UNNotificationServiceExtension {
                 ImageDownloader.default.download(URLRequest(url: url), completion: { [weak self] response in
                     guard let _ = self else { return }
                     switch response.result {
-                    case .failure(let error):
-                        os_log(.info, log: .debug, "%{public}s[%{public}ld], %{public}s: download image %s fail: %s", ((#file as NSString).lastPathComponent), #line, #function, url.debugDescription, error.localizedDescription)
```

Comment:
I'm wondering if this might actually be worth keeping...but I'm not sure how valuable it is given logs from the `NotificationService` are most likely rarely looked at anyways.

Reply:
Exactly, that's what I thought in most places: We don't do anything with these information. Logging an Error to Debug isn't very helpful. That's why I removed this (in several places)